limesurvey/limesurvey.py

```diff
@@ -286,6 +316,11 @@ def studio_submit(self, data, suffix=""):  # pylint: disable=unused-argument
         self.display_name = data.get("display_name")
         self.survey_id = data.get("survey_id")
         self.limesurvey_url = data.get("limesurvey_url")
+        if self.limesurvey_url:
+            self.limesurvey_url = self.limesurvey_url.rstrip("/")
```

Comment:
What is the purpose of this?

Reply:
removing the trailing slash in case people add https://limesurvey.com/ instead of https://limesurvey.com

Reply:
can you try that out? :)

Reply:
perfect, I tried it. I find it useful